src/components/ChatInput/ChatInput.js

```diff
@@ -90,15 +95,56 @@ const ChatInput = () => {
     }
     await RCInstance.sendAttachment(event.target.files[0]);
   };
+  const getAllChannelMembers = async () => {
+    const channelMembers = await RCInstance.getChannelMembers();
+    setMembers(channelMembers.members);
+  };
 
   useEffect(() => {
     if (editMessage.msg) {
       messageRef.current.value = editMessage.msg;
     }
   }, [editMessage]);
+  useEffect(() => {
+    getAllChannelMembers();
+  }, []);
 
   return (
     <Box m="x20" border="2px solid #ddd">
+      {showMembersList ? (
```

Comment:
can we please separate this out to its own component?

Reply:
Yes. Thanks for suggesting changes!